integration_tests/tests/apply/apply.cy.ts

```diff
@@ -696,8 +705,8 @@ context('Apply', () => {
 
       // Then the application should be submitted to the API
       cy.task('verifyApplicationUpdate', application.id).then(requests => {
-        expect(requests).to.have.length(36)
-        const requestBody = JSON.parse(requests[35].body)
+        expect(requests).to.have.length(37)
```

Comment:
Idly wondering if we can do something about these magic numbers here - is there maybe a way we can count the number of pages or something?

Reply:
Yep, will fix in a subsequent PR